addon/components/img-lazy.js

```diff
@@ -143,7 +159,7 @@ function isLoaded(img) {
 
 function fetchImage(url) {
   return new Promise((resolve, reject) => {
-    const img = new Image
+    let img = new Image
```

Comment:
This change seems unnecessary, even if I personally tend to use `let` in newer projects, I'd rather have this in a separate PR doing only "cosmetic" changes.